plugins/CoreAdminHome/CustomLogo.php

```diff
@@ -196,18 +196,22 @@ private function uploadImage($uploadFieldName, $targetHeight, $userPath)
         list($width, $height) = getimagesize($file);
         switch ($_FILES[$uploadFieldName]['type']) {
             case 'image/jpeg':
-                $image = imagecreatefromjpeg($file);
+                $image = @imagecreatefromjpeg($file);
```

Comment:
It throws a warning if the file is invalid, which might be shown as notification.